apps/api/src/app/shared/services/cache/cache.service.ts

```diff
@@ -1,9 +1,9 @@
-import Redis from 'ioredis';
-import { ConnectionOptions } from 'tls';
 import { Logger } from '@nestjs/common';
 import { QUERY_PREFIX } from './key-builders/shared';
 
-const STORE_CONNECTED = 'ready';
```

Comment:
As the connected status would vary from provider to provider we move the responsibility to the InMemoryProviderService.